component/client/core/ClientCore.tsx

```diff
@@ -110,7 +110,7 @@ class ClientCore extends React.Component<any, ClientCoreState> {
   }
 
   private handleInit(data): void {
-    if (data.initialized) {
+    if (data.initialized && data.initialized.newValue !== undefined) {
```

Comment:
Ensure that `data.initialized` exist and `newValue` is defined. 

This change occurs in several different places so that the data with just the `oldValue` field set does not set the state's value to `undefined`.